src/components/institution_table/InstitutionTableUtils.tsx

```diff
@@ -0,0 +1,291 @@
+ */
+const columnWidths = {
+  id: '10%',
+  name: '25%',
+  domains: '15%',
+  signingOfficials: '25%',
+  updateUser: '15%',
+  updateDate: '10%'
+}
+
+const baseTemplateStyle = {
+  margin: '1rem 2%'
+}
+
+// Template used for a skeleton loader that shows a loading state for the table.
+export const tableHeaderTemplate = (
+    <>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.id}}}>ID</div>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.name}}}>Institution</div>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.domains}}}>Domains</div>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.signingOfficials}}}>Signing Officials</div>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.updateUser}}}>Update User</div>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.updateDate}}}>Updated On</div>
+    </>
+);
+
+// Template used for a skeleton loader that shows a loading state for the table.
+export const tableRowLoadingTemplate = (
+    <>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.id}}} className="text-placeholder"/>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.name}}} className="text-placeholder"/>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.domains}}} className="text-placeholder"/>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.signingOfficials}}} className="text-placeholder"/>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.updateUser}}} className="text-placeholder"/>
+      <div style={{...baseTemplateStyle, ...{width: columnWidths.updateDate}}} className="text-placeholder"/>
+    </>
+);
+
+export const tableStyles = {
+  baseStyle: {
+    fontSize: '1.5rem',
+    display: 'flex',
+    padding: '1rem 2%',
+    backgroundColor: 'white',
+    border: '1px solid #DEDEDE',
+    margin: '0.5% 0'
+  },
+  columnStyle: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: '1rem',
+    color: '#7B7B7B',
+    fontSize: '1.6rem',
+    fontWeight: 'bold',
+    letterSpacing: '0.2px',
+    textTransform: 'uppercase',
+    backgroundColor: 'B8CDD3',
+    border: 'none'
+  },
+};
+
+/**
+ * The primary configuration for the columns in the institution table.
+ */
+export const columnConfig: ColumnConfig = {
+  id: {
+    label: 'ID',
+    cellStyle: {width: columnWidths.id},
+    cellDataFn: (row: Institution) => row.id,
+    sortable: true,
+    sortValueFn: (row: Institution) => row.id,
+  },
+  name: {
+    label: 'Institution',
+    cellStyle: {width: columnWidths.name},
+    cellDataFn: (row: Institution) => {
+      if (row) {
+        return <Link
+            to={{pathname: `/admin_manage_institutions/institutions/${row.id}`}}
+            style={{
+              overflow: 'hidden',
+              whiteSpace: 'nowrap',
+              textOverflow: 'ellipsis',
```

Comment:
I'm guessing this isn't working as expected.  See my screenshot.

Reply:
I'll take another look, I think this is copied over from the original style.